samples/word/02-paragraphs/insert-formatted-text.yaml

```diff
@@ -1,4 +1,5 @@
-id: word-basics-insert-formatted-text
+order: 1
+id: word-paragraphs-insert-formatted-text
 name: Insert formatted text
 description: Shows how to use formatting objects as well as applying pre-built styles.
```

Comment:
We've edited other snippet descriptions to remove the phrase "Shows how to..." -- should the same thing be done for this description?